rt/main.tf

```diff
@@ -0,0 +1,24 @@
+# Route table
+
+resource "aws_route_table" "rt" {
+  count  = "${(var.vgw_prop + 1 % 2) * var.rt_count}"
```

Comment:
Feature flag is unfortunately necessary, as the length of a calculated value (vgw_ids) cannot be determined during planning.